packages/compass-e2e-tests/helpers/commands/scroll-to-virtual-item.ts

```diff
@@ -0,0 +1,53 @@
+import type { CompassBrowser } from '../compass-browser';
+
+export async function scrollToVirtualItem(
+  browser: CompassBrowser,
+  itemsSelector: string,
+  targetSelector: string
+): Promise<boolean> {
+  let found = false;
+
+  let lastId: string;
+
+  // Start by scrolling down, then scroll back up. This is necessary in case we
```

Comment:
Isn't it possible to just move back to `y=0` first and then just scroll down?

Reply:
how?

Reply:
that much I don't know, of course, that's why I was asking :). I would imagine you could use js of the window as a last resort:

``` js
browser.execute(function() {
    document.querySelector(containerSelector).scrollTop = 0;
});
```

Just to be clear, this is a nit, looks good to me as it is, feel free to merge.

Reply:
I thought about that. There's an element in the dom somewhere that seems to have the total height so that it would render scrollbars and that's probably the one that gets scrolled. Doesn't have a name or id or anything so we'd have to try and find it by some "parent of something" or "child of something" trickery, then access it with browser.execute() and passing code snippets..

Can probably work, but feels even more hacky to me and then the code has to know even more about the exact inner workings of the component and it works even less like the way a human would scroll to the card..

I'm on the fence, but this seems to work, so could go with this for now and just change it later.